modules/vaos/spec/services/eps/access_token_store_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Eps::AccessTokenStore do
+  subject { described_class }
+
+  let(:access_token) { 'test-access-token' }
+  let(:token_type) { 'jwt_bearer' }
+  let(:redis_key) { "eps-access-token:#{token_type}" }
+  let(:cache_data) { { token_type:, access_token: } }
+  let(:token_store_client) { subject.new(token_type:, access_token:) }
+
+  describe '#save' do
+    it 'saves the value in cache' do
+      token_store_client.save
```

Comment:
this can be added in a `before` block. that way, you just need 1 before block for all specs

Reply:
Addressed in [8a42abd](https://github.com/department-of-veterans-affairs/vets-api/pull/19307/commits/8a42abdde300b9c181770687232c14a36c272a5b)